chain/ethereum/build.rs

```diff
@@ -1,4 +1,5 @@
 fn main() {
+    println!("cargo:rerun-if-changed=proto");
```

Comment:
Witchcraft! For my education, where are the docs on this?

Reply:
https://doc.rust-lang.org/cargo/reference/build-scripts.html